dtfs-central-api/src/v1/controllers/tfm/amendments/tfm-get-amendments.controller.js

```diff
@@ -51,7 +50,7 @@ const findAllAmendmentsByFacilityId = async (facilityId) => {
     if (!ObjectId.isValid(facilityId)) {
       throw new Error('Invalid facility Id');
     }
-    const collection = await db.getCollection(MONGO_DB_COLLECTIONS.TFM_FACILITIES);
+    const collection = await TfmFacilitiesRepo.getCollection();
```

Comment:
I dont think there's a good alternative to using these `aggregate` pipelines but I wish they were also in the repo & all the mongodb stuff could be hidden

Reply:
FEATURE ENVY